Resources/Textures/Objects/Weapons/Melee/stunbaton.rsi/meta.json

```diff
@@ -26,6 +26,24 @@
                 ]
             ]
         },
+        {
+            "name": "equipped-BELT",
+            "directions": 4,
+            "delays": [
+                [
+                    1.0
+                ],
+                [
+                    1.0
+                ],
+                [
+                    1.0
+                ],
+                [
+                    1.0
+                ]
```

Comment:
if you want remove delays

Reply:
I just left the format the same as i'm going to refactor those .jsons at some point and these are temp since I want clothing on these items to point to the overlay rsi